src/dataviews/dataview-model-base.js

```diff
@@ -322,6 +322,10 @@ module.exports = Model.extend({
     return sourceAnalysis && sourceAnalysis.get('type');
   },
 
+  getLayerName: function () {
+    return this.layer.get('layer_name');
```

Comment:
@matallo it seems that `layer` is not required (we don't enforce setting it in the constructor) so you might want to change this to `return this.layer && this.layer.get('layer_name')`